nmap: add -O example (#4400)
tldr-pages/tldr: pages/common/nmap.md

> Some features only activate when Nmap is run with privileges.
> More information: <https://nmap.org>.

- If Nmap throws a root privileges error use it with `sudo` or `admin rights cmd`

> - To check if the IP address `192.168.1.101` is up, with it’s operating system info. It may show RTTVAR logs (for TCP) for a long time:

Reviewer: Perhaps this will read better?
```suggestion
- Check if a given IP address is up, and attempt to get the remote operating system:
```

Author: This is better:
```suggestion
- Check if an IP address is up, with OS info:
```

Reviewer: Hrm, I'm not keen on shortening `OS info` there, and it's also ambiguous what it does with it. What about this?
```suggestion
- Check if an IP address is up, and guess the remote host's operating system:
```

> `nmap -O 192.168.1.101`

Reviewer: Missing `{{token_syntax}}`?
```suggestion
`nmap -O {{ip_or_hostname}}`
```

> - To scan the Network for devices (including routers) and get info on their operating system:

Reviewer: Perhaps this will read better?
```suggestion
- Scan a block of IP addresses, and additionally attempt to guess the operating system on any hosts found:
```

> `sudo nmap -sS -O 192.168.1.0/24`

Reviewer: Missing token syntax
```suggestion
`sudo nmap -sS -O {{192.168.1.0/24}}`
```

Author: Check this out:
```suggestion
`sudo nmap -sS -O {{IP/port}}`
```

Reviewer: Hrm. It's not actually a port - it's a CIDR block number. We could do this?
```suggestion
`sudo nmap -sS -O {{ip_address}}/{{CIDR_block}}`
```
.....but it's not obvious what you're supposed to put there.  To that end, I think a concrete example would be best for this example:
```suggestion
`sudo nmap -sS -O {{192.168.1.0}}/{{24}}`
```

> Some features only activate when Nmap is run with privileges.
> More information: <https://nmap.org>.

> # nmap

Reviewer 2: Two headers?

Author: fixed

> network scanning tool
> More information: <https://nmap.org/>.

> - If Nmap throws a root privileges error use it with `sudo` or `admin rights cmd`

Reviewer 2: We don't include this kind of message in our pages.

Reviewer: Generally our format is to have a description, followed by an example.

Also, we try to keep pages focused on the command in question, and avoid explaining generic shell concepts such as `sudo`. We do have a convention to prefix commands that require sudo with `sudo` in the example, but in this case as far as I know `nmap` doesn't require `sudo` to function (it just takes longer).

To that end, I'd suggest we remove this line.

Author: Done!
